app/models/validations/sales/household_validations.rb

```diff
@@ -61,4 +77,8 @@ def person_economic_status_refused?(economic_status)
   def person_is_child?(relationship)
     relationship == "C"
   end
+
+  def tenant_is_economic_child?(economic_status)
+    economic_status == 9
+  end
```

Comment:
how do we feel about endless method syntax for one liners like these?
`def tenant_is_economic_child?(economic_status) = economic_status == 9`
probably golf code for no benefit since they are not widely used as far as I know
I think I'm just in the habit of one liners for small things coming over from C# land

Reply:
Hm, I don't don't mind too much either way. I personally find it a bit easier to read when it's on separate lines and it's currently consistent with what we have everywhere else, but if you can convince the team that this is worth a refactor (cause this happens across the whole codebase), I can live with that!

Reply:
I am very much not sure I prefer it, but I was excited to learn about this syntax as I was not previously aware of it